### ISSUE TITLE
New plugin name rule breaks builds and affects other packages

### ISSUE THREAD

**Issue by dragonjet** [senecajs/seneca#733]
```
Error: seneca: The plugin name string cannot be empty and must be alphanumeric (matching /^[a-zA-Z][a-zA-Z0-9_]*$/), and cannot be longer than 1024 characters. Name was '....'.
```
We are using `.` as plugin name in our production and that killed all our services on scaling after this rule change. I believe breaking changes should only be implemented in major versions so like us who use `^3.` in our `package.json` won't be affected. This might not even have been intentional on your part, I see more of an oversight than anything, so I'd like to let you know there are those using `.` in their plugin names. Please consider this, and other characters - as well as how you publish breaking changes.

Thanks

**Comment by matiger** [comment]
The package` seneca-amqp-transport` is using the `(package.json).version` as `PLUGIN_TAG`. Obviously there is a `.` in there as well (`2.2.0` right now). Since the tag has the same regex I am getting an error message too now.

**Comment by rjrodger** [comment]
this is now fixed in 3.8.2
